### ISSUE TITLE
Easy way to limit to one camera?

### ISSUE THREAD

**Issue by peterbaker** [gkoehler/LogiCircleDownloader#9]
Is there any obvious/easy way to limit the downloading to one camera? I suppose I could hardcode the ID I'm seeing in the log, but is there a more maintainable way to do this?

**Comment by gkoehler** [comment]
That sounds like a great feature request! I'll leave this issue open in case someone wants to implement it. Maybe a command-line option? `--camera-id`

**Comment by peterbaker** [comment]
Yeah, command line would be cool, or even just in the config file.
